hoomd/md/pair/pair.py

```diff
@@ -323,7 +326,7 @@ class SLJ(Pair):
         nlist (`hoomd.md.nlist.NList`): Neighbor list
         r_cut (float): Default cutoff radius (in distance units).
         r_on (float): Default turn-on radius (in distance units).
-        mode (str): Energy shifting/smoothing mode
+        mode (str): Energy shifting mode
```

Comment:
Some of these attributes end with periods and others do not. Try to make that consistent across all force parameter docstrings. Specifically I see periods missing from `mode` and `nlist` (``nlist (`hoomd.md.nlist.NList`): Neighbor list``).

Also it's unclear to me why the word "smoothing" was deleted here but kept in the description below.
```suggestion
        mode (str): Energy shifting/smoothing mode.
```